packages/tse-backend/src/modules/exchange-data/exchange-data.mapper.ts

```diff
@@ -0,0 +1,88 @@
+import { AutomapperProfile, InjectMapper } from '@automapper/nestjs';
+import { Injectable } from '@nestjs/common';
+import { createMap, forMember, mapFrom, Mapper } from '@automapper/core';
+import {
+  GetMultipleTickerPricesCommand,
+  GetMultipleTickerPricesDto,
+  GetOHLCVCommand,
+  GetOHLCVDto,
+  GetSupportedSymbolsCommand,
+  GetSupportedSymbolsDto,
+  GetTickerPriceCommand,
+  GetTickerPriceDto,
+  GetTickersCommand,
+  GetTickersDto,
+} from './model/exchange-data.model';
+
+@Injectable()
+export class ExchangeDataProfile extends AutomapperProfile {
+  constructor(@InjectMapper() mapper: Mapper) {
+    super(mapper);
+  }
+
+  override get profile() {
```

Comment:
What's that?, why do we need to ovverride it?

Reply:
https://automapperts.netlify.app/docs/nestjs#automapperprofile